sonar-cxx-plugin/src/main/java/org/sonar/plugins/cxx/coverage/CoberturaParser.java

```diff
@@ -37,6 +37,14 @@
  * {@inheritDoc}
  */
 public class CoberturaParser implements CoverageParser {
```

Comment:
CoverageParser is an interface so I can't move it there.  Do you want me to create a base class and put baseDir there?  If so, do you have a preference for a name?